### ISSUE TITLE
COCO pre-training size

### ISSUE THREAD

**Issue by e-bug** [uclanlp/visualbert#17]
Hi! Could you share the size of pre-training data?
I saw that you extend the training set with part of the validation set.

**Comment by liunian-harold-li** [comment]
I think it was 100k images with 5 captions each. I used MSCOCO train+val data, which is conventional.
